lib/notes.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+class Notes {
+  constructor(object) {
+    const { action, payload } = object.command;
+    this.action = action;
+    this.payload = payload;
+
+  }
+  execute() {
```

Comment:
under here you should be doing some to tell that if the action is add go to this.add.